package/yast2-live-installer.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Jul 28 13:03:03 UTC 2016 - cyberorg@opensuse.org
+
+- correct_live_* scripts are removed from kiwi config
+  these scripts are required to get yast2-live-installer function,
+  the scripts belong in this package
```

Comment:
this make sense, but maybe have it as yast clients? this way even more yast functionality can be used.